test/gtest/ucs/test_time.cc

```diff
@@ -53,27 +53,26 @@ UCS_TEST_SKIP_COND_F(test_time, get_time,
 }
 #endif
 
-UCS_TEST_F(test_time, timerq) {
-    static const int TIMER_ID_1  = 100;
-    static const int TIMER_ID_2  = 200;
+void init_timerq(ucs_timer_queue_t *timerq) {
```

Comment:
can we move this inside test_time class?

Reply:
I'm not sure we have to make this change, checking